geodetic_utils/include/geodetic_utils/geodetic_conv.h

```diff
@@ -0,0 +1,208 @@
+#ifndef GEODETIC_CONV_H_
+#define GEODETIC_CONV_H_
+
+#include "math.h"
+#include <Eigen/Dense>
+
+namespace geodetic_conv {
+// Geodetic system parameters
+static const double semimajor_axis = 6378137;
+static const double semiminor_axis = 6356752.3142;
+static const double first_eccentricity_squared = 6.69437999014 * 0.001;
+static const double second_eccentricity_squared = 6.73949674228 * 0.001;
+static const double flattening = 1 / 298.257223563;
```

Comment:
switch to constexpr (http://stackoverflow.com/questions/13346879/const-vs-constexpr-on-variables).
And if you use google style guide, constants are named as `kConstantName` (http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Constant_Names), but this is up to you :).